### DISCUSSION TITLE
Limit Number of Descendant Generations

### DISCUSSION THREAD

**Discussion by AvgustKuznetsov** [PeWu/topola-viewer#132] [Ideas]
I'm using Topola Viewer to display a tree from WikiTree. Would it be possible to have a setting to lower the number of displayed descendant generations? Otherwise some of the trees with many descendants become extremely wide and hard to read.

It seems that this constant in _src/datasource/wikitree.ts_ is responsible for setting the limit:
`const descendantGenerationLimit = 5;`

**Comment by allangould** [comment]
I would be interested in being able to display a subset of my tree.  I am using Topola Viewer from gramps (on Windows; GRAMPS: AIO64-5.1.4-1 ) as "Interactive family tree".  It does the job very nicely on the whole database, but it would be nice to select a subset, for example, 3 generations of descendants from person ID.  Just a thought.  I realise I could export all the records I want into a new gramps database then run "Interactive family tree" but that is a little cumbersome and having the ability to select in topola viewer would be very nice!

**Comment by angelhrod** [comment]
I just signed up to suggest something like this. I think it would be really useful to have an option to limit the number of ancestor generations displayed, as the tree becomes very wide and hard to read.
